### PR TITLE
Adding support for DSCP configuration

### PR DESCRIPTION
**Pull request by karthiksundaravel**
Allow the user to do DSCP mapping for network objects Interface and SriovPF. The feature needs the handling of netlink messages. The pyroute2 module is used in the encode/decode of the netlink messages.
The persistence of the DSCP configurations are taken care by the os-net-config-dcb-config service. The user configurations are preserved in /var/lib/os-net-config/dcb_config.yaml

The details of the DSCP configurations could be read anytime with the cli.
``os-net-config-dcb --show``

The new DSCP configurations can be applied with cli using ``os-net-config-dcb --config <config.yaml>``

The sample config.yaml for updating DCB only is available at etc/os-net-config/samples/dcb_config_sample.yaml

(cherry picked from commit 3c11aef6d3dde83217a47b8a6d0e5f7f8018b2b2)

### CODE REVIEW

**Comment by vcandapp** [comment]
recheck

**Review by vcandapp — CHANGES_REQUESTED** [review]

File: etc/os-net-config/samples/dcb_config_sample.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# For reconfiguring the DCB, the below template could be used.
```
[the remark this answers is not in the document]
Comment (karthiksundaravel) [reply etc/os-net-config/samples/dcb_config_sample.yaml:1]: I'll raise another PR on master

[the remark this answers is not in the document]
Comment (karthiksundaravel) [reply etc/os-net-config/samples/dcb_config_sample.yaml:1]: Its handled by https://github.com/os-net-config/os-net-config/pull/46. And when its merged, I'll raise another PR for backport to stable/wallaby

**Review by vcandapp — APPROVED** [review]
lgtm

**Pull request merged**